dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/bnd.bnd

```diff
@@ -20,7 +20,7 @@ src: \
 fat.project: true
 
 tested.features: \
-	jacc-1.5, expressionLanguage-4.0, appsecurity-4.0, servlet-5.0, cdi-3.0, enterpriseBeansLite-4.0, appAuthorization-2.1
+	jacc-1.5, expressionLanguage-4.0, appsecurity-5.0, servlet-6.0, cdi-3.0, enterpriseBeansLite-4.0, appAuthorization-2.1
```

Comment:
Since it is only EE10 now I assume that jacc-1.5 expressionLanguage-4.0 and cdi-3.0 are not actually tested so should not be here.

Reply:
Changed